config/bolt/demo.js

```diff
@@ -4,7 +4,7 @@ configure({
   ],
   sources: [
     source('amd', 'ephox.imagetools.demo', '../../src/demo/js', mapper.hierarchical),
-    source('amd', 'ephox', '../../lib/run/depend', mapper.flat)
+    source('amd', 'ephox', '../../src/main/js', mapper.flat)
```

Comment:
This isn't required, it's covered by importing `prod.js`. lib/run/depend is where our internal dependencies went, if they aren't being used anymore then `config/dent/depend.js` can be deleted as well.